sdk/eventhub/eventhubs-checkpointstore-blob/README.md

```diff
@@ -1,19 +1,140 @@
 
 ## Getting started
 
-## Key concepts
+### Install the package
+
+Install the Azure Event Hubs Checkpoint Store Blob library using npm
+
+`npm install @azure/eventhubs-checkpointstore-blob`
+
+**Prerequisites**: You must have an [Azure subscription](https://azure.microsoft.com/free/), an
+[Event Hubs Namespace](https://docs.microsoft.com/en-us/azure/event-hubs/) to use this package, and a [Storage account](https://docs.microsoft.com/en-us/azure/storage/blobs/storage-blobs-introduction)
+If you are using this package in a Node.js application, then use Node.js 8.x or higher.
+
+### Configure Typescript
+
+TypeScript users need to have Node type definitions installed:
+
+```bash
+npm install @types/node
+```
+
+You also need to enable `compilerOptions.allowSyntheticDefaultImports` in your tsconfig.json. Note that if you have enabled `compilerOptions.esModuleInterop`, `allowSyntheticDefaultImports` is enabled by default. See [TypeScript's compiler options handbook](https://www.typescriptlang.org/docs/handbook/compiler-options.html) for more information.
+
+### Key concepts
+
+- **Checkpointing** is a process by which readers mark or commit their position within a partition event sequence. Checkpointing is the responsibility of the consumer and occurs on a per-partition basis within a consumer group. This responsibility means that for each consumer group, each partition reader must keep track of its current position in the event stream, and can inform the service when it considers the data stream complete.
+
+  If a reader disconnects from a partition, when it reconnects it begins reading at the checkpoint that was previously submitted by the last reader of that partition in that consumer group. When the reader connects, it passes the offset to the event hub to specify the location at which to start reading. In this way, you can use checkpointing to both mark events as "complete" by downstream applications, and to provide resiliency if a failover between readers running on different machines occurs. It is possible to return to older data by specifying a lower offset from this checkpointing process. Through this mechanism, checkpointing enables both failover resiliency and event stream replay.
+
+- Event Processor based application consists of one or more instances of `EventProcessor` which have been
+  configured to consume events from the same Event Hub and consumer group. They balance the
+  workload across different instances by distributing the partitions to be processed among themselves.
+  They also allow the user to track progress when events are processed using checkpoints.
+  A checkpoint is meant to represent the last successfully processed event by the user from a particular
+  partition of a consumer group in an Event Hub instance.
```

Comment:
We dont need to talk about what a checkpoint is here again after we added 2 big paragraphs for the same before :)